apps/src/blockly/addons/contextMenu.js

```diff
@@ -184,6 +185,36 @@ const registerUnshadow = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(unshadowOption);
 };
 
+const registerKeyboardNavigation = function() {
+  const keyboardNavigationOption = {
+    displayText: function(scope) {
+      const displayText = scope.workspace.keyboardAccessibilityMode
+        ? msg.blocklyKBNavOff()
+        : msg.blocklyKBNavOn();
+      return displayText;
```

Comment:
nit: the `displayText` variable feels unnecessary here -- returning the value from the ternary here is probably clear enough

Reply:
That's fair. There are other context menu options in this file with more complex values where the use of the variable was justified. I think I primarily kept it here to maintain a pattern. No need though if it just reads as more complicated!